docs/.vuepress/theme/styles/quirks.styl

```diff
@@ -0,0 +1,10 @@
+/* 
+  A temporary stylesheet containing various
+  styles for random elements of the documentation.
+*/
+
+
+/* Fix the font color of the <pre> console in the saving-data.md page */
```

Comment:
I would prefer to add a class `example-color-white`
Then it is available to use `::: example #example1 .example-color-white `